Packs/Base/ReleaseNotes/1_12_4.md

```diff
@@ -0,0 +1,4 @@
+#### Scripts
+##### SanePdfReports
+- Multiple bug fixes
```

Comment:
If you do not explain what was wrong and how it was fixed, keeping this line is worse than deleting it.

Reply:
@yanyo87 - You can use the docs https://xsoar.pan.dev/docs/documentation/release-notes#bug-fixes-examples for examples on how bug fix RN should look like.

Reply:
i'm familiar with that, 
but we have more than 50 prs 
![image](https://user-images.githubusercontent.com/61918543/121061648-c8bcbc00-c7cc-11eb-8071-c8f2bb9c4310.png)

Reply:
We should state that the new docker version includes the new version of sane-reports, which includes new reports and dashboarding functionalities / bug fixes.
We should obviously not write all changes, and state only the major ones with a reference to the sane-reports repo (which is an open source repository).